guidelines/style-guidelines.md

```diff
@@ -0,0 +1,162 @@
+## Table of contents
+
+1. [How we write](#how-we-write)
+2. [Grammar and punctuation](#grammar-and-punctuation)
+3. [Links](#links)
+4. [Lists](#lists)
+5. [Numbers](#numbers)
+6. [Symbols, currency and abbreviations](#Symbols,-currency-and-abbreviations)
+7. [Common terms](#common-terms)
+
+
+## How we write
+
+#### Voice and tone
+
+MetaBrainz speaks in a friendly, direct and easily understandable manner. We:
+* use plain, simple, language when possible
+* use short sentences
+* use an active voice (avoid the passive tense).
+
+#### Spelling
+
+We use US English.
+
+#### Active and passive writing 
+
+Active writing mentions the subject (the person or thing 'doing' the action) first in the sentence (for example, inflation pushed up house prices).
+
+Passive writing mentions the object (the person or thing 'receiving' the action) first. Passive sentences often include 'by' (for example, house prices were pushed up by inflation).
+
+Active writing is more direct and should be used most of the time. We use short, clear sentences.
+
+
+## Grammar and punctuation
+
+#### Exclamation marks
+
+We don't use exclamation marks in most instances - for instance, we don't use exclamation marks in documentation or page content.
+
+#### Colon and semi-colons
+
+We avoid these. When possible we’ll write 2 sentences instead, or separate the clauses using a dash (with a space on either side).
```

Comment:
Why? :( I mean, dashes are ok, but I really don't like this obsession with it being a replacement for everything somehow. https://thenarrativearc.org/blog/2020/2/4/epic-grammar-battle-semicolon-versus-em-dash for example. Sometimes text just flows better with a colon or a semicolon.

Reply:
Short sentences are always better, particularly for people who might be reading in a second language, or have other reading difficulties. Though I am a chronic long sentencer-er... I try and keep myself in check. So I think it's a good reminder that a colon/dash could often just be two sentences instead. I stand behind that bit.

Otherwise I don't think it's about _never_ using a colon or a semi colon, just being consistent when you are choosing between the two. We could easily swap this to be 'avoid dashes'?

For me it is quite noticeable if someone is writing with a lot of dashes - it's pretty distinct - or if they're using a lot of semicolons; also very distinct. So either way it's nice to offer guidance if someone is wondering which to go with.

Reply:
Yeah, trying to have shorter sentences is not a bad idea. Maybe this should be "Colons, semicolons and dashes": "If it makes sense, write two shorter sentences rather than using these" then?

Reply:
In French, dashes are _apposition_, that can also be written with (round) parenthesis or even with comma (but it's more oral style, then).